### ISSUE TITLE
GAC NullPointerException Error | player can't join the server

### ISSUE THREAD

**Issue by GoodLucky777** [PowerNukkit/PowerNukkit#937]
# 🔌 Plugin compatibility issue

### ▶ Steps to Reproduce

1. Install GAC v1.3.8
2. Run the server and you will see the NPE error.
3. Player can't never join the server, no login process.

### ✔ Expected Behavior

NPE error not occured.

### ❌ Actual Behavior

NPE error and player can't join the server.

### 📋 Debug information

* PowerNukkit version: 1.4.0.0-PN-SNAPSHOT (9d8a942d27a7ede511a09adaade6691a5d23b36a)
* Debug link: -
* Timings link (if relevant): -

### 💢 Crash Dump, Stack Trace and Other Files

```
[main] FATAL - Could not load plugin
java.lang.NullPointerException: null
	at cz.creeperface.nukkit.gac.aq.a(Reflect.kt:1036) ~[?:?]
	at cz.creeperface.nukkit.gac.V.<init>(GACNetworkInterface.kt:30) ~[?:?]
	at cz.creeperface.nukkit.gac.am.a(Hacks.kt:21) ~[?:?]
	at cz.creeperface.nukkit.gac.GTAnticheat.<init>(GTAnticheat.kt:36) ~[?:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_275]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_275]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_275]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_275]
	at java.lang.Class.newInstance(Class.java:442) ~[?:1.8.0_275]
	at cn.nukkit.plugin.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:59) ~[powernukkit.jar:?]
	at cn.nukkit.plugin.PluginManager.loadPlugin(PluginManager.java:122) [powernukkit.jar:?]
	at cn.nukkit.plugin.PluginManager.loadPlugins(PluginManager.java:308) [powernukkit.jar:?]
	at cn.nukkit.plugin.PluginManager.loadPlugins(PluginManager.java:159) [powernukkit.jar:?]
	at cn.nukkit.plugin.PluginManager.loadPlugins(PluginManager.java:151) [powernukkit.jar:?]
	at cn.nukkit.plugin.PluginManager.loadPlugins(PluginManager.java:147) [powernukkit.jar:?]
	at cn.nukkit.Server.<init>(Server.java:675) [powernukkit.jar:?]
	at cn.nukkit.Nukkit.main(Nukkit.java:192) [powernukkit.jar:?]
[main] FATAL - Could not load plugin "GAC"
```

### 💬 Anything else we should know?

-

**Comment by GoodLucky777** [comment]
But GAC 1.3.4 new raknet version was working without error and login problem.

**Comment by joserobjr** [comment]
Thank you for the report!

The current GAC that is hosted on https://cloudburstmc.org/resources/gac.119/ is outdated and won't work even on regular Nukkit itself. This happens because GAC is doing some forbidden stuff by accessing private internal code by reflection, and that code has been refactored to support the newest Minecraft protocol.

The good news is that the GAC devs has already updated the plugin, but they didn't released a version update to the page, they instead added a link to their dropbox with the working JAR file.

Click on the highlighted word on the page to download GAC instead of the orange Download button:
![image](https://user-images.githubusercontent.com/3679022/107879651-dc0fe880-6eb8-11eb-80e4-2dbc49133cf6.png)

I did a simple test with that version and GAC was able to load, I was able to join and do some simple stuff.
